app/lib/features/attachments/widgets/attachment_item.dart

```diff
@@ -43,49 +42,55 @@ class AttachmentItem extends ConsumerWidget {
       decoration: BoxDecoration(
         color: containerColor,
         borderRadius: BorderRadius.circular(8),
-        border: Border.all(color: borderColor),
+        border: Border.all(color: greyColor),
       ),
       child: ListTile(
-        leading:
-            mediaState.mediaLoadingState.isLoading || mediaState.isDownloading
-                ? loadingIndication()
-                : attachmentLeadingIcon(attachmentType),
+        leading: attachmentLeadingIcon(attachmentType),
         onTap: () => attachmentOnTap(
           ref,
           context,
           attachmentType,
           mediaState,
         ),
-        onLongPress: canEdit
-            ? () => openRedactContentDialog(
+        title: title(context, attachmentType),
+        trailing: Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            if (mediaState.mediaFile == null &&
+                attachmentType != AttachmentType.link)
+              mediaState.mediaLoadingState.isLoading || mediaState.isDownloading
+                  ? loadingIndication()
+                  : IconButton(
+                      onPressed: () => ref
+                          .read(
+                              attachmentMediaStateProvider(attachment).notifier)
+                          .downloadMedia(),
+                      icon: Icon(
+                        Icons.download,
+                        color: Theme.of(context).primaryColor,
+                      ),
+                    ),
+            if (canEdit)
+              IconButton(
```

Comment:
didn't we say we put this into three dots into a submenu? I find the strong red delete marker taking too much attention ...

Reply:
Haven't did work regarding Edit Attachment stuff, so that's why putted delete button directly. I will add more button along with the Edit and Delete menu.